packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -424,6 +424,11 @@ export interface IButtonStyles {
    */
   description?: IStyle;
 
+  /**
+   * Style for the description text if applicable (for compound buttons.)
+   */
+  secondaryText?: IStyle;
```

Comment:
secondary text is here, but isn't being used in button.

If the idea is to replace description with secondaryText, lets start supporting both (in BaseButton.tsx and style files) so that we can deprecate description

Reply:
Whoops, sorry no I forgot to remove this. Am adding vr-tests now

Reply:
On second thought I am not sure. Without this addition I am unable to add this property to ContextualMenu.styles.ts (since this derives from IMenuStyles which falls under ButtonTypes.ts (namely, `IMenuItemStyles extends IButtonStyles`). Is this fine? Or how do you suggest adding it properly to just effect ContextualMenuItems?